### ISSUE TITLE
Decoding Camera Image

### ISSUE THREAD

**Issue by NoraKrbr** [shaqian/flutter_realtime_detection#3]
I followed the article you wrote about this on Medium and everything has worked for me except the section “Decoding CameraImage”. Could you tell me where I should include the code snippet that you provided (feed-ios.mm) in the native iOS code?

**Comment by shaqian** [comment]
Hi,

The native code is already included in fluter [tflite](https://github.com/shaqian/flutter_tflite) package. You just need to call the methods (`Tflite.runModelOnImage()` etc) from flutter code. 

Thanks,
Qian

**Comment by NoraKrbr** [comment]
Hi Qian,

thanks four you answer. Turns out I did something wrong when I tried to incorporate some parts of your code into my project. When I just deploy your app on my phone, it works perfectly.
I have more of a general question about the tflite plugin, which I didn't figure out. Is it possible to import a custom model with it that I built with tensorflow, and use that one for real-time object detection?

Thanks,
Nora
